docs/UsersGuide/best/hub-pattern.md

```diff
@@ -16,5 +16,132 @@ ARINC 653 Channels, High-speed hardware buses between nodes, and UARTs between n
 
 ## Generic Hub
 
-There is now a standard implementation of the hub pattern. The [GenericHub](../api/c++/html/svc_generic_hub_component.html) is an
-implementation of the hub pattern that passes through F´ ports and `Fw::Buffer`s.
\ No newline at end of file
+There is now a standard implementation of the hub pattern. The 
+[GenericHub](../api/c++/html/svc_generic_hub_component.html) is an implementation of the hub pattern 
+that passes through F´ ports and `Fw::Buffer`s.
+
+## Detailed Example Generic Hub Setup
+
+A sample F´ project featuring a two-deployment hub setup can be found on [fprime-community](@FIXME).
```

Comment:
I might emphasize the example even more - maybe it's own section in the document?